tests/test_imageIo2.py

```diff
@@ -54,7 +54,7 @@ def testIo(self):
             image = Image(self.cols, self.rows)
             for x in range(0, self.cols):
                 for y in range(0, self.rows):
-                    image.set(x, y, x + y)
+                    image[x, y, afwImage.PARENT] = x + y
```

Comment:
see comment in `test_background.py`. There's no need to include the origin since `PARENT` is the default value.